packages/arco-lib/src/components/Table.tsx

```diff
@@ -178,6 +217,24 @@ export const Table = implementRuntimeComponent({
     direction: undefined,
   });
   const [filterRule, setFilterRule] = useState();
+  const [columns, setColumns] = useState<ColumnProperty[]>([]);
+
+  const handleResize = (
```

Comment:
Can we wrap the resize logic to a component? `Table.tsx` is big and complex enough.

Reply:
The table dragging logic is deeply coupled with the column property and is not easily separated